components/social-share.jsx

```diff
@@ -0,0 +1,46 @@
+var React = require('react');
+
+var SocialShareButton = React.createClass({
+    propTypes : {
+        networkType: React.PropTypes.string.isRequired,
+        iconClassName: React.PropTypes.string.isRequired
+    },
+    render : function () {
+        return (
+            <a href="#" className={"btn icon-only btn-social " + this.props.networkType }>
```

Comment:
`href="#"` means "navigate to the top of this page" under HTML5 rules, so if this is a button, it should probably be a `<button>` with styling to make it look like how we want it to look, rather than be a link. If there is no url, it's not a link =)